src/components/Picklist/styled/input.js

```diff
@@ -36,7 +36,13 @@ const PickerInput = attachThemeAttrs(styled(StyledInput))`
             box-shadow: none;
         `}
     `}
-
+    ${props =>
+        props.variant === 'bare' &&
+        `
+        :hover {
+            box-shadow: ${props.shadows.shadow_10};
```

Comment:
when `:hover` add  to the `background-color: `${props.palette.background.main};`